qutebrowser/browser/qutescheme.py

```diff
@@ -135,6 +135,11 @@ def data_for_url(url):
     Return:
         A (mimetype, data) tuple.
     """
+    # normalize path and strip redundant trailing slashes
```

Comment:
I don't think this comment is needed - it says the same the code in the line below says :wink:

Reply:
Yeah, I agree. I'll remove this.